day-12/src/components/PokemonRow.jsx

```diff
@@ -4,26 +4,25 @@ import { Button } from "@material-ui/core";
 
 import PokemonType from "../pokemonType";
 
-const PokemonRow = ({ pokemon, onClick }) => (
```

Comment:
Can we drop the reformatting to tabs here as it's not part of the real change?

Reply:
Sure